views/profile.html

```diff
@@ -6,43 +6,25 @@
 		<div class="container">
 			<h1><%- heading %></h1>
 
-			<!-- Account status -->
-			<% if (message.length > 0) { %>
-			<div class="alert alert-warning"><%= message %></div>
-			<% } %> <% if (user.verified) { %>
-			<div class="alert alert-success">Account status: verified</div>
-			<% } else { %>
-			<div class="alert alert-danger">
-				Please verify your account with Slack.
-				<a href="/auth/slack"
-					><img
-						class=""
-						alt="verify in with Slack"
-						height="40"
-						width="172"
-						src="https://platform.slack-edge.com/img/sign_in_with_slack.png"
-						srcset="
-							https://platform.slack-edge.com/img/sign_in_with_slack.png    1x,
-							https://platform.slack-edge.com/img/sign_in_with_slack@2x.png 2x
-						"
-				/></a>
-			</div>
+			<!-- alerts -->
+			<% if (errorMessage.length > 0) { %>
+			<div class="alert alert-danger"><%= errorMessage %></div>
+			<% } %>
+			<% if (warningMessage.length > 0) { %>
+			<div class="alert alert-warning"><%= warningMessage %></div>
 			<% } %>
-
 			<!-- Update Account -->
-			<form action="/profile?_method=PUT" method="post" class="">
+			<form>
 				<div class="form-group">
 					<label for="email">Email</label>
 					<input type="email" class="form-control" name="email" value="<%= user.email %>" disabled />
```

Comment:
Are we going to use the email field anymore? It struck me as odd when I logged in via slack and the profile page showed a disabled, blank email. I think it should be removed.

Reply:
Agree

Reply:
this is a bug. adding "identity.email" to `scope` list in `auth.js` resolves.